### ISSUE TITLE
What is self.N0 for?

### ISSUE THREAD

**Issue by leom97** [mbrossar/denoise-imu-gyro#10]
Hello, may I ask why are `self.N0` predictions from the network removed, when computing the loss during training? My guess is this should account for the initial padding, but why have you set `self.N0 = 5`, always?
Thanks in advance!

**Comment by RXvo** [comment]
Hello, do you still have the corresponding optimization parameters and data set of this code? The author's links have all failed. Thank you！

**Comment by scarLiu** [comment]
> Hello, do you still have the corresponding optimization parameters and data set of this code? The author's links have all failed. Thank you！

Hello, do you solve the problem now? I have encountered with this trouble www

**Comment by leom97** [comment]
@RXvo , @scarLiu No, I have never downloaded the pretrained models and dataset. @mbrossar would you be so kind to provide us with an updated link? Thanks in advance :)
